### PR TITLE
Add private benchmarking IP range to local list

### LINKED ISSUE #104: RFC 2544 benchmarking IP addresses should be considered private
The `198.18.0.0/15` benchmarking IP range reserved by [RFC 2544](https://www.rfc-editor.org/rfc/rfc2544) is not globally routable, so it should be considered private similar to the logic for CGNAT (`100.64.0.0/10`) in #92. The more recent [RFC 6890](https://www.rfc-editor.org/rfc/rfc6890) confirms that this range is still reserved, and it's also marked as not globally routable in the [IANA IPv4 Special-Purpose Address Registry](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml).

We're considering using this as a low-conflict private IP range for a use case where traffic never leaves the same physical machine. It'd be great if the range could be added to PNA in order to protect local services running within the subnet.

Happy to make a pull request if the idea is accepted.

### PR DESCRIPTION
**Pull request by kdrag0n**
Closes #104.

### CODE REVIEW

**Comment by kdrag0n** [comment]
> * if your contribution does not concern a normative part of a specification, or is editorial in nature (e.g. fixing typos or examples), you don't need to do anything

I assume this change is small enough that there's no need to do anything here?

File: index.src.html [hunk 1]
```diff
@@ -409,6 +409,12 @@ <h3 id="ip-address-space-heading">IP Address Space</h3>
         <td>[[RFC6598]]</td>
         <td>[=IP address space/local=]</td>
       </tr>
+      <tr>
+        <td>`198.18.0.0/15`</td>
+        <td>Benchmarking</td>
+        <td>[[RFC2544]]</td>
+        <td>[=IP address space/local=]</td>
+      </tr>
```
Comment (letitz) [inline index.src.html:417]: Mind moving this one row lower? I tried to keep the table sorted lexicographically by (IP address space, IP address range).

**Review by letitz — CHANGES_REQUESTED** [review]
Thanks very much for sending this in!

Unfortunately the change is to a normative part of the spec, so the IPR issue must be resolved.

File: index.src.html (hunk 1 above)
Comment (kdrag0n) [reply index.src.html]: Done, although I don't quite understand the sorting order (e.g. why 198.18 should be above 192.168).

Also reached out to get the IPR issue resolved. Will post an update when that's done.

Comment (letitz) [reply index.src.html]: Aha, you're right. I read 192.18... And the link-local address range right below is not sorted correctly. Oh well. Good enough as is.

**Comment by yoavweiss** [comment]
Hey @kdrag0n !

Can you [join](https://www.w3.org/community/wicg/) the WICG to appease the IPR bots?

File: index.src.html (hunk 1 above)
Comment (kdrag0n) [reply index.src.html]: Ah, that makes more sense. Moved it below the 192.168 range.

**Comment by kdrag0n** [comment]
> Hey @kdrag0n !
> 
> Can you [join](https://www.w3.org/community/wicg/) the WICG to appease the IPR bots?

Thanks @yoavweiss! Unfortunately that didn't work.

Screenshot 2023-05-25 at 11 24 02 AM

**Comment by kdrag0n** [comment]
Looks like it works now, sorry for the noise. Just needed some time to propagate.

**Review by letitz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**